src/api/spoonacular.py

```diff
@@ -474,6 +474,55 @@ def search_ingredients(
     return ingredients
 
 
+def get_recommended_recipes(
+    offset: int = 0,
+    limit: int = 10,
+) -> list:
+    """
+    Returns a list of randomly recommended recipes.
+
+    Args:
+        Number as the number of desired recipe results
+
+    Returns:
+        A list of JSON-encoded recipes.
+        Each recipe object consists of the following values:
+            - id (int): The ID of the recipe.
+            - name (str): The display name of the recipe.
+            - image (str): The URL of the image for the recipe.
+
+    Raises:
+        UndefinedApiKeyException: If the Spoonacular API key is undefined.
+        SpoonacularApiException: If there was a problem completing the request.
+    """
+
+    params = {"apiKey": get_api_key()}
+
+    params["offset"] = offset
+    params["number"] = limit
+
+    data = None
+    try:
+        data = api_get_json(
+            SPOONACULAR_API_ROOT_ENDPOINT + f"recipes/random?",
```

Comment:
I dont think the '?' is necessary in the url
```suggestion
            SPOONACULAR_API_ROOT_ENDPOINT + "recipes/random",
```